### ISSUE TITLE
[Bug] Android image buttons display some of the accessibility readout visually when given x:Name

### ISSUE THREAD

**Issue by patchandthat** [xamarin/Xamarin.Forms#9293]
### Description

On Android, have a button with an image, accessibility name/helptext, and no button text.  If the button is then given `x:Name` property, the accessibility readout will be rendered over the button.

Happens when the button renderer is derived from FastRenderers.ButtonRenderer, issue is not present in AppCompat.ButtonRenderer

### Steps to Reproduce

See reproduction link. Image selected uses transparency.

`<Button
                    x:Name="BiometricButton"
                    ImageSource="touch_id"
                    BackgroundColor="Transparent"
                    AutomationProperties.IsInAccessibleTree="True"
                    AutomationProperties.HelpText="image button voiceover help text"
                    AutomationProperties.Name="image button voiceover name"/>`

### Expected Behavior
Only the button image should be visible on screen

### Actual Behavior
The button image and some of the ContentDescription is visible on screen

### Basic Information

- Version with issue: 4.4
- Last known good version: N/A
- IDE: VS 2019
- Platform Target Frameworks: 
  - iOS:  N/A
  - Android: 9 & 10
  - UWP:  N/A
- Android Support Library Version: N/A
- Nuget Packages:
- Affected Devices:

### Screenshots

Observed:
![droid_btn_1](https://user-images.githubusercontent.com/9745915/72979845-1a7bc680-3dd1-11ea-9e75-6da1b80ec3cd.png)

Expected:
![droid_btn_2](https://user-images.githubusercontent.com/9745915/72978861-1353b900-3dcf-11ea-8209-11d674eea0a8.png)

### Reproduction Link

Branch: image-button-accessibility
https://github.com/patchandthat/jubilant-potato/tree/image-button-accessibility

### Workarounds

Setting height/width requests will cause this text to be truncated, but not hidden completely.

Alternatively use `AppCompat.ButtonRenderer`, but that suffers from the issue described in #9090

**Comment by PureWeen** [comment]
@patchandthat this looks like it's been fixed with the latest release

Can you try the latest 4.5 pre and let me know? I was able to reproduce with your sample but then after I upgraded the text went away

**Comment by patchandthat** [comment]
@PureWeen Yes, this looks fixed to me.  Thanks.
